Add instructions for making block attribute required with custom validator (#2309)
ibexa/documentation-developer: docs/content_management/pages/page_block_validators.md

``` yaml hl_lines="16-18"
[[= include_file('code_samples/page/custom_block_validator/config/packages/page_blocks.yaml', 0, 1) =]][[= include_file('code_samples/page/custom_block_validator/config/packages/page_blocks.yaml', 3, 20) =]]
```

### Custom required validator

> By default, only `not_blank` and `not_blank_richtext` cause a block attribute to be marked as required.

Reviewer: Nice to have a list of validators automatically triggering the "required" mark. It justifies a new section by itself. I would also add a new paragraph separator:

```suggestion
By default, only `not_blank` and `not_blank_richtext` validators mark a block attribute as required.

```

If you create a custom validator `custom_not_blank` with attribute-specific logic, you can add extend the `AttributeType` class with a Symfony form type extension to ensure that the attribute is also considered required:

> ``` php

Reviewer: I would highlight `$builder->setRequired(true);`:
```suggestion
``` php hl_lines="16"
```

``` yaml hl_lines="16-18"
[[= include_file('code_samples/page/custom_block_validator/config/packages/page_blocks.yaml', 0, 1) =]][[= include_file('code_samples/page/custom_block_validator/config/packages/page_blocks.yaml', 3, 20) =]]
```

### Custom required validator

By default, only `not_blank` and `not_blank_richtext` validators mark a block attribute as required.

> If you create a custom validator `custom_not_blank` with attribute-specific logic, you can extend the `AttributeType` class with a Symfony form type extension to ensure that the attribute is also considered required:

Reviewer: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/ensure-insure-assure
```suggestion
If you create a custom validator `custom_not_blank` with attribute-specific logic, you can extend the `AttributeType` class with a Symfony form type extension to make sure that the attribute is also considered required:
```
